lib/event_store_client/subscriptions.rb

```diff
@@ -3,16 +3,17 @@
 module EventStoreClient
   class Subscriptions
     def create(subscriber, event_types)
-      event_types.each do |type|
-        subscription = subscriptions[type.to_s] || Subscription.new(type: type, name: service)
-        subscription.subscribers |= [subscriber]
-        create_subscription(subscription) unless @subscriptions.key?(type.to_s)
-        @subscriptions[type.to_s] = subscription
+      subscription = Subscription.new(subscriber, event_types: event_types, service: service)
+      connection.join_streams(subscriber.class.name, subscription.observed_streams)
+      unless @subscriptions.detect { |sub| sub.name == subscription.name }
+        create_subscription(subscription)
```

Comment:
Hm.. not sure just question. Is it correct we join_streams in line 7 and also inside create_subscription?